tools/ml-audio-content-moderation/app/angular/src/app/toxicity-text-section.ts

```diff
@@ -0,0 +1,6 @@
+export class ToxicityTextSection {
```

Comment:
would this be better as an interface instead of as a class?
https://www.typescriptlang.org/docs/handbook/interfaces.html